src/PageContent/RequestPickup/RecurringPickupRequest.js

```diff
@@ -130,10 +130,13 @@ class RecurringPickupRequest extends Component {
         return formIsValid;
     }
 
-    handleChange(field, e) {   
-        let fields = this.state.fields;
-        fields[field] = e.target.value; 
-        this.setState({fields});
+    handleChange(field, e) {
+        var val = e.target.value;
+        this.setState((prevState) => {
```

Comment:
Is there any particular reason why you changed this to an arrow function?

Reply:
While testing there was something that failed I thought this could be causing it. `setState()` is an async function and if the state update depends on the previous state, then this arrow function style is how you're suppose to do it. I think this didn't end up being the cause of the error, but i still left this change in cuz i think this is still more correct.